lib/main_containers.pm

```diff
@@ -208,6 +226,7 @@ sub load_container_tests {
                 # Common openQA image tests
                 load_image_tests_podman($run_args) if (/podman/i);
                 load_image_tests_docker($run_args) if (/docker/i);
+                load_host_tests_in_k8s($run_args) if (/k8s/i);
```

Comment:
I see that it is similar to two lines above so most probably it is just my bad perl knowledge but I can not understand what it is going on in this line and same as two lines above . I understand that `/k8s/i` is regex but with what we comparing it ? 0_o

Reply:
I can explain... long story short... you can have `CONTAINER_RUNTIME=podman,docker` so you run podman tests and docker tests in the same job. There is a loop in this code:
`foreach (split(',\s*', $runtime)) {`
so, basically `if (/podman/i);` means if the current element of the loop is `podman`... maybe not so trivial and elegant, but it works pretty well :)

Reply:
my question is pure perl grammar , I was expecting something like `$_` somewhere in this if clause , but there is nothing 0_o so how I can guess against what `/podman/i` query is running ?

Reply:
perl magic :D

Reply:
:see_no_evil:  - I am out :)